firmware/include/runtest.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+# Copyright (c) 2019 Red Hat, Inc. All rights reserved. This copyrighted material
+# is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General
+# Public License v.2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+
+# Include rhts environment
+. /usr/bin/rhts-environment.sh || exit 1
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+# Include cki library
+. ../../cki_lib/libcki.sh
+
+# Task parameters
+# DeBug - Set to non-zero value to enable debugging
+# FwtsGitRemote - git repository
+# FwtsGitBranch - git branch that will be used
+
+
+: ${DeBug:=0} # Set to non-zero value to enable debugging
```

Comment:
I guess this is a comment more for the original maintainer than you, but probably the comment on this line should be dropped, as it's already right above.